core/server/data/meta/title.js

```diff
@@ -12,7 +12,7 @@ function getTitle(data, root, options) {
     options = options ? options : {};
 
     if (pagination && pagination.total > 1) {
-        pageString = ' (Page ' + pagination.page + ')';
+        pageString = pagination.pageString || ' (Page ' + pagination.page + ')';
```

Comment:
The meta title is also called when using `{{ghost_head}}`. That means this solution only works when i use the `{{meta_title}}` helper.

Reply:
I don't understand. When I look to the output of <code>{{ghost_head}}</code>, I don't see any 'page' string that could be translated. 
Is it in <code>ghost_head.js</code> that the meta title is also called upon? Can you show me where, so I can investigate, cause I can't find it. 😞

Reply:
I looked into this and noticed the `{{ghost_head}}`-helper only uses `title.js` for generating the meta-tags (for twitter, facebook, etc...). These meta-tags are not printed in a paged context. See line 155 in `ghost_head.js`: 
<code>if (**!**_.includes(context, 'paged') && useStructuredData) {
                    head.push('');
                    head.push.apply(head, finaliseStructuredData(metaData));
                    head.push('');</code>

Knowing this, I think my solution doesn't form any problem. When translation is needed (in a paged context), the translated string is not printed in the meta-tags.

Unless, this printing of the meta-tags should happen in a paged context. The if-statement was added a long time ago in this commit by @cobbspur: https://github.com/TryGhost/Ghost/commit/a48458b73dc1c8bf0a730d33e3461cad512c61f6 (rewritten here: 5102637b8e23c0d33704406003a2a81d08752f37), to fix bug #5146.

Can I remove this check, so meta-tags get printed in a paged context? When I tested it here, it worked perfectly (error pages have their own template now). If this is possible, I can move the translation logic.

@kirrg001 or @cobbspur, can you tell me what to do? 😃

Reply:
Yeah you are right. The `{{ghost_head}}` only generates the meta description. Sorry for confusion 🙈